avd/vlans.yml

```
@@ -6,3 +6,5 @@ vlans:
     name: Twenty
   - 30:
     name: Thirty
+  - 40:
+    name: Forty
```

we should create vlan 41 instead
```suggestion
    name: Forty-one
```

---

no this neads to be vlan 40